src/dev/i18n/extract_default_translations.js

```diff
@@ -38,15 +40,56 @@ function addMessageToMap(targetMap, key, value) {
   targetMap.set(key, value);
 }
 
-export async function extractDefaultTranslations(inputPath) {
+async function getPluginsPaths(inputPaths) {
+  const availablePaths = Object.values(config.paths);
+  const pathsForExtraction = [];
```

Comment:
__question:__ looks like this can contain duplicates, should it be `Set` instead?